docs/decisions/0002-mark-fuzzy-entries.rst

```diff
@@ -0,0 +1,99 @@
+Mark Transifex invalid entries as fuzzy
+#######################################
+
+Context
+*******
+As of the `OEP-58`_, the Transifex GitHub App is used to sync Translations.
+These translations are validated by `LexiQA`_, built-in Transifex quality
+checks, and human reviewers. During this synchronization process, the
+`Transifex GitHub App`_ creates pull requests to this repository. The
+translations in the pull requests are then validated by ``msgfmt`` in CI.
+
+Problem
+*******
+There are times when the translations being synchronized fail ``msgfmt``
+validation. This prevents the pull requests from being merged.
+
+
+Design Decision
+***************
+
+A GitHub Actions workflow will be implemented to mark invalid entries in
+synchronized ``.po`` files as ``fuzzy``. This will update pull requests
+created by the `Transifex GitHub App`_.
+
+To ensure a safe and reliable workflow, the following workflows will be
+combined into one single workflow with multiple jobs:
+
+#. Run ``python-tests.yml`` to validate the python code
+#. Then, run ``validate-translation-files.yml`` which performs the following:
+
+   #. Validate the po-files using ``msgfmt``
+   #. Notify the translators about the invalid entries via the preferred
+      communication channel (Slack, Transifex, GitHub)
+   #. Edit the po files to mark invalid entries as ``fuzzy``, so it's
+      excluded from ``.mo`` files
+   #. Revalidate the files
+
+#. Commit the fixes
```

Comment:
I'm not sure if "fixes" is the best word here, as the workflow isn't fixing entries, it's just marking broken ones as fuzzy.

Maybe just
```suggestion
#. Commit the updated entries
```